pkg/deployment/deployment.go

```diff
@@ -607,3 +607,17 @@ func (builder *Builder) WithToleration(toleration coreV1.Toleration) *Builder {
 
 	return builder
 }
+
+// WithSchedulerName configures a scheduler to process pod's scheduling.
+func (builder *Builder) WithSchedulerName(schedulerName string) *Builder {
```

Comment:
Please move this function up closer to other With* functions. Public functions have to be above private funcs

Reply:
done